### PR TITLE
Complex sqrt: fix for #20

### PR DESCRIPTION
**Pull request by joslloand**
Fix for: https://github.com/supercollider-quarks/MathLib/issues/20

### CODE REVIEW

File: classes/various/extComplex.sc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+//---------------------------------------------------------------------
+//	MathLib: Some mathematical extensions to SuperCollider.
+//
+// 	Extension: Complex
+//
+//---------------------------------------------------------------------
+
+
+/* J Anderson, 2019 */
++ Complex {
```
Comment (telephon) [inline classes/various/extComplex.sc:22]: This should go into the common library. Could you make a pull request there?

**Review by telephon — CHANGES_REQUESTED** [review]

File: classes/various/extComplex.sc (hunk 1 above)
Comment (joslloand) [reply classes/various/extComplex.sc:22]: @telephon, yep, you're right!

Given that I'm _really_ looking to solve https://github.com/supercollider-quarks/MathLib/issues/20, and I'm hoping to simplify local roll-out for dev version of ATK, how do you feel about:

1. A PR refactoring uses in `Polynomial` from `Complex.sqrt` to `Complex.pow(0.5)`
2. Submitting `Complex.sqrt` addition to the common library

This route would mean that https://github.com/supercollider-quarks/MathLib/issues/20 is quickly solved before then common library solution becomes available.

Comment (telephon) [reply classes/various/extComplex.sc:22]: That's good!

**Comment by joslloand** [comment]
Closing this PR now, with reference to newly created PR #32

**Pull request closed without merging**